api/src/wfl/module/covid.clj

```diff
@@ -111,6 +112,8 @@
                       :sink     (sink/load-sink! tx workload)}]
     (as-> workload $
       (select-keys $ workload-metadata-keys)
+      ;; need to deserialize the watchers array
+      (update $ :watchers #(mapv all/parse-watcher %))
```

Comment:
The comment can go; it just confuses.  Consider:
```
;; need to parse the watchers sequence
(update $ :watchers (partial mapv all/deserialize-watcher))
```
Call it either parse or deserialize, but not both.